client/my-sites/email/email-providers-stacked-comparison/provider-cards/utils.tsx

```diff
@@ -13,6 +15,28 @@ export enum IntervalLength {
 	MONTHLY = 'monthly',
 }
 
+export const formattedPriceWithInterval = (
```

Comment:
I don't think this function is being called or imported from any of the other files in this PR. I think we can remove it.